File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
import java.util.*;
import java.lang.*;

class Solution {
    boolean solution(String s) {
        
        int numOfP = 0;
        int numOfY = 0;
        int len = s.length();
        
        for(int i=0; i<len; i++){
            String tmp = s.substring(i, i+1);
            if(tmp.equals("p") || tmp.equals("P")){
                numOfP += 1;
            }
            else if(tmp.equals("y") || tmp.equals("Y")){
                numOfY += 1;
            }
        }

        if(numOfP == numOfY){
            return true;
        }
        return false;
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    cntP = 0
    cntY = 0
    
    for c in s:
        tmp = c.upper()
        if tmp == 'P':
            cntP += 1
        elif tmp == 'Y':
            cntY += 1
    
    if cntP == cntY :
        return True

    return False